profiles/common/modules/features/multisite_settings/multisite_settings_core/multisite_settings_core.install

```diff
@@ -80,6 +73,8 @@ function multisite_settings_core_update_7122() {
 }
 
 /**
+ * Implements hook_update_N().
```

Comment:
No need for this: https://www.drupal.org/docs/develop/coding-standards/api-documentation-and-comment-standards#updateN